logs/field.go

```diff
@@ -257,4 +278,12 @@ const (
 
 	// StringerType corresponds to fmt.Stringer
 	StringerType
+
+	EndType
 )
+
+func (ft FieldType) String() string {
+
+	return []string{"invalid", "int", "int64", "string", "bool", "time.Duration", "[]string", "error", "any", "stringer", "endtype"}[ft]
```

Comment:
1. It is not efficient - create []string slice for every function call
2. It is bad readable and bad checkable (if we reorder fields in definition)

What about use switch and case for every variant?